enhancements/ingress/wildcard-routing.md

```diff
@@ -0,0 +1,189 @@
+---
+title: wildcard-routing-policy
+authors:
+  - "@danehans"
+reviewers:
+  - "@openshift/api-reviewers"
+  - "@openshift/sig-network-edge"
+  - "@danehans"
+
+approvers:
+  - "@openshift/api-approvers"
+creation-date: 2020-03-19
+last-updated: 2020-03-19
+status: implementable
+see-also:
```

Comment:
This enhancement builds on the `route-admission-policy` enhancement, so it would make sense to link to that.

Reply:
done